src/jemalloc.c

```diff
@@ -1497,6 +1498,21 @@ malloc_conf_init_helper(sc_data_t *sc_data, unsigned bin_shard_sizes[SC_NBINS],
 				CONF_HANDLE_BOOL(
 				    opt_prof_experimental_use_sys_thread_name,
 				    "prof_experimental_use_sys_thread_name")
+				if (CONF_MATCH("prof_time_resolution")) {
+					if (CONF_MATCH_VALUE("default")) {
+						opt_prof_time_resolution = prof_time_resolution_default;
```

Comment:
That 8 space shiftwidth/tabstop sure do make it difficult to get into 80 characters. I've refactored all my variable names so that "resolution" has been contracted to "res" so that lines fit in the 80 chars.